### DISCUSSION TITLE
Zoom on continuously updating LineSeries<DateTimePoint>

### DISCUSSION THREAD

**Discussion by aj-viper** [Live-Charts/LiveCharts2#138] [Q&A] [answered]
I have a LineSeries with a DateTimePoint. Before I zoom in the charts x axis moves because I am removing the front item and adding an item to the back. When I use the zoom feature, it zooms into what I want to see and stops scrolling the time axis (as I expect). When I zoom back out to see all of the data, I would expect the x axis to continue scrolling as data is added but it does not. Is there a way I can reset the zoom status once I zoom out all the way so that it continues scrolling?

**Comment by mark-monteiro** [comment, marked as answer]
You should just need to reset the `MinLimit` and `MaxLimit` properties on the zoomed axis to null and the axis will reset to automatically calculate its bounds.

See the documentation here: https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/components/1.7.axes.md#maxlimit-and-minlimit-properties
